src/server/game/Quests/QuestTracker.cpp

```diff
@@ -0,0 +1,117 @@
+/*
+ * This file is part of the WarheadCore Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "QuestTracker.h"
+#include "DatabaseEnv.h"
+#include "World.h"
+#include "TaskScheduler.h"
+#include <tuple>
+#include <vector>
+
+namespace
+{
+    std::vector<std::tuple<uint32 /*ID*/, uint32 /*CharLowGuid*/, std::string /*Hash*/, std::string /*Rev*/>> _questTrackStore;
+    std::vector<PreparedStatement*> _queueStmt;
+    TaskScheduler scheduler;
+}
+
+QuestTracker* QuestTracker::instance()
+{
+    static QuestTracker instance;
+    return &instance;
+}
+
+void QuestTracker::InitSystem()
+{
+    if (!sWorld->getBoolConfig(CONFIG_QUEST_ENABLE_QUEST_TRACKER))
+    {
+        sLog->outString(">> System disabled");
```

Comment:
let's use a more meaningful message here, so we know where it comes from. Example:

```
">> The QuestTracker System is disabled"
```